bloom/github.py

```diff
@@ -99,6 +99,16 @@ def do_github_post_req(path, data=None, auth=None, site='api.github.com'):
     return response
 
 
+def json_loads(resp):
```

Comment:
@gemarcano could you please add a docstring to this function detailing its utility as a wrapper for python2/3 compatibility?